### PR TITLE
chore: bump opencv version 

### PR DESCRIPTION
**Pull request by montekki**
## Purpose

Bumps `opencv` to a new version and uses `QrCodeDetector`.

### CODE REVIEW

File: .github/workflows/android-app.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   build:
     name: Build
-    runs-on:            ubuntu-22.04
+    runs-on:            ubuntu-latest
```
Comment (sergejparity) [inline .github/workflows/android-app.yml:13]:
```suggestion
    runs-on:            ubuntu-20.04
```
I would suggest to use `ubuntu-20.04` explicitly, as it is expected that `ubuntu-latest` will start pointing to `ubuntu-22.04` soon. https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/

File: .github/workflows/android-app.yml [hunk 2]
```diff
@@ -75,7 +75,7 @@ jobs:
 
   debug:
     name: Build unsigned APKs
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```
Comment (sergejparity) [inline .github/workflows/android-app.yml:78]:
```suggestion
    runs-on:            ubuntu-20.04
```
I would suggest to use `ubuntu-20.04` explicitly, as it is expected that `ubuntu-latest` will start pointing to `ubuntu-22.04` soon. https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/

Comment (sergejparity) [inline .github/workflows/android-app.yml:78]:
```suggestion
    runs-on: ubuntu-20.04
```
I would suggest to use `ubuntu-20.04` explicitly, as it is expected that `ubuntu-latest` will start pointing to `ubuntu-22.04` soon. https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/

Comment (montekki) [reply .github/workflows/android-app.yml:78]: I would actually want to switch to `22.04` but even with that the build would not work it looks like

**Review by krodak — APPROVED** [review]

**Pull request merged**